uprotocol/uattributes.proto

```diff
@@ -46,38 +46,43 @@ message UAttributes {
     // Message type (publish, request, response)
     UMessageType type = 2;
 
+    // The source (address) of the message. For published events this is the producers publish topic
+    // for requests it is the calling uE respose topic (who sent the request). For responses this is the
+    // method URI from the uService. 
+    UUri source = 3;
```

Comment:
I hope you [know what you are doing](https://protobuf.dev/programming-guides/dos-donts/#reuse-number), shifting the identifiers ;-)